test/e2e/data/infrastructure-docker/main/clusterclass-quick-start.yaml

```
@@ -440,21 +483,38 @@ spec:
       - containerPath: "/var/run/docker.sock"
         hostPath: "/var/run/docker.sock"
 ---
+apiVersion: infrastructure.cluster.x-k8s.io/v1beta1
+kind: DockerMachinePoolTemplate
+metadata:
+  name: quick-start-default-worker-machinepooltemplate
+  labels:
+    InfraMachineTemplate.machinePool.label: "InfraMachineTemplate.machinePool.labelValue"
```

```suggestion
    InfraMachinePoolTemplate.machinePool.label: "InfraMachinePoolTemplate.machinePool.labelValue"
```
similar for the annotation